sds_data_manager/lambda_code/SDSCode/instruments/l1c_codice.py

```diff
@@ -1,5 +1,6 @@
 """Lambda runtime code that triggers off of arrival of data into S3 bucket.
 """
+import json
 import logging
 import os
```

Comment:
What is the difference between these lambdas? Can they just be the same?

Reply:
They are the same. But eventually each instrument will modify their own lambda. Right now they exist only for the purposes of demonstration.

Reply:
What kind of differences will exist in instrument-specific lambdas? Could we have one base lambda that everyone uses and maybe change that for specific instrument changes? I'm not sure what differences there will be across instruments, but I would prefer to try very hard to keep the instrument processing steps the exact same for all instruments.